requirements.txt

```diff
@@ -1,5 +1,6 @@
-# This can not be installed from pip, on Fedora you need to install rpm-python{,3}
-#rpm-python
+# This can not be installed from pip, on Fedora you need to install python3-rpm.
+# https://github.com/rpm-software-management/rpm/tree/master/python/rpm
+#python3-rpm
```

Comment:
OK I will wait until F24 EOL. maybe we need to wait for 1 month.

> https://fedoraproject.org/wiki/Fedora_Release_Life_Cycle
> Fedora 24 will be maintained until 1 month after the release of Fedora 26. 

By the way I could not find `rpm3-python` pip package on my F25.

```
$ rpm -q python2-rpm
python2-rpm-4.13.0.1-1.fc25.x86_64

$ rpm -q python3-rpm
python3-rpm-4.13.0.1-1.fc25.x86_64

$ /usr/bin/pip list | grep rpm
rpm-python (4.13.0.1)

$ /usr/bin/pip3 list | grep rpm
rpm-python (4.13.0.1)
```